src/games/strategy/engine/framework/ProcessRunnerUtil.java

```diff
@@ -101,13 +102,11 @@ public static void exec(final List<String> commands) {
       // we need to read the input stream to prevent possible
       // deadlocks
       final Thread t = new Thread(() -> {
-        try {
-          while (s.read() >= 0) {
-            // just read
-          }
-        } catch (final IOException e) {
-          ClientLogger.logQuietly(e);
+        Scanner scanner = new Scanner(s);
```

Comment:
The benefit of sending the output, is that I could read debug information like system.out messages and exception stacktraces even after starting those processes.
Abou the blocking: Since we can't interact with it afterwards vie command line, and we know it's triplea we are starting, it's clear that most output will come in lines.
I don't think that something will block there